stories/2-Card.stories.js

```diff
@@ -130,3 +133,93 @@ export const NoteCard = () => (
+        </div>
+        <DoneChecking style={{width: "80%"}} src={done_checking}/>
+      </HintStyle>
+      <HintStyle style={{border: "none", minWidth: "15.5%", width: "15.5%"}}>
+        <PrimaryWithText />
+      </HintStyle>
+    </React.Fragment>
+  </Background>
+);
+
+export const OnboardingCard = () => (
+  <Background style={{height: 1024, padding: "10px"}}>
+    <ContextStyle style={{minWidth: "65%", width: "65%"}}>
+      <div className="div" style={{minHeight: 400}}>
+        <h1 style={{fontWeight: "bold", color: "black"}}>Get Started</h1>
+        <p style={{color: "black", minWidth: "100%", width: "100%"}}>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Facilisis leo vel fringilla est. Orci a scelerisque purus semper eget duis. Dolor sit amet consectetur adipiscing elit duis tristique. Amet venenatis urna cursus eget nunc scelerisque viverra mauris in. Sapien eget mi proin sed libero enim sed faucibus turpis. Non nisi est sit amet facilisis magna. Arcu cursus vitae congue mauris rhoncus. Augue mauris augue neque gravida in fermentum et sollicitudin. Malesuada fames ac turpis egestas sed tempus urna et pharetra. Leo urna molestie at elementum eu facilisis sed odio morbi.
+        </p>
+        <br/>
+        <p style={{color: "black"}}>
+abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz</        p>
+      </div>
+      <Button primary onClick={() => console.log("create goals!")}>Create Goals</Button>
+    </ContextStyle>
+  </Background>
+);
+
+export const RepoDetailsCard = () => {
+  //this logic grabs the icons and puts the icons and text into an arr
+  //of objects to be mapped over
+  const repoDetailDiv = () => {
+    const octoface = getIconByName("octoface");
+    const star = getIconByName("star");
+    const contributions = getIconByName("git-commit");
+    const openContributions = getIconByName("repo-pull");
+
+    const detailInfo = [{icon: octoface, text: "120 contributors"},
+      {icon: star, text: "14K stars"}, {icon: contributions, text: "0 contributions"}, {icon: openContributions, text: "0 open contributions"}];
+    //map over arr of objects from line 192
+    return detailInfo.map(item => {
+      //retain an indiv div with just the contributors, stars, contributions and open
+      return (
+        <FlexCenter style={{width: "100%", flexWrap: "wrap"}}>
+          <div style={{width: "30px", color: "grey"}}><Octicon  verticalAlign="middle" icon={item.icon}/></div>
+          <div style={{fontSize: "15px", color: "grey"}} >{item.text}</div>
+        </FlexCenter>
+      );
+    });
+  };
+  //this is the render for the actual component
+  return (
+    <Background style={{height: 1024}}>
+      <Card>
+        <Flex style={{justifyContent: "space-around"}}>
+          {/* TODO: un-hard code this...
+          funcs are not appropriate React children*/}
```

Comment:
Thanks, this can be addressed in the implementation.

Reply:
👍